citest/gcp_testing/google_session.py

```diff
@@ -73,7 +74,8 @@ def __init__(
   def logout(self):
     """Log out of the current authenticated Google session.
     """
-    raise NotImplementedError()
+    self.__session.get("https://www.google.com/accounts/Logout")
+    self.__session.cookies.clear()
```

Comment:
Is clearing the cookies ok here, or does that change behavior to an unauthenticated call? Could this result in future calls being unauthenticated (and passing with some different behavior for anonymous clients) vs failing because the provided cookie is invalid?
Just curious. Otherwise LGTM